### PR TITLE
feat: add developers res hub link to library

### PR DESCRIPTION
**Pull request by oriole-pub**

### CODE REVIEW

File: site/src/data/archives.data.ts [hunk 1]
```diff
@@ -2,6 +2,16 @@ import { Archive, ArchiveCategory } from '~/types/library';
 import { contributorsData } from '~/data/contributors.data';
 
 export const archives: Array<Archive> = [
+  {
+    id: '74a6440375df7b5c',
+    title: 'Nervos Network Developers Resources Hub',
+    description: 'The most authoritative official guide',
```
Comment (Flouse) [inline site/src/data/archives.data.ts]:
```suggestion
    description: 'A collection of developer resources from Nervos Network community',
```

File: site/src/data/archives.data.ts [hunk 2]
```diff
@@ -2,6 +2,16 @@ import { Archive, ArchiveCategory } from '~/types/library';
 import { contributorsData } from '~/data/contributors.data';
 
 export const archives: Array<Archive> = [
+  {
+    id: '74a6440375df7b5c',
+    title: 'Nervos Network Developers Resources Hub',
```
Comment (Flouse) [inline site/src/data/archives.data.ts]:
```suggestion
    title: 'Nervos Network Developer Resources Hub',
```

**Review by Flouse — CHANGES_REQUESTED** [review]

File: site/src/data/archives.data.ts (hunk 1 above)
Comment (oriole-pub) [reply site/src/data/archives.data.ts]: Sorry, I overlooked this. 😭

File: site/src/data/archives.data.ts (hunk 2 above)
Comment (oriole-pub) [reply site/src/data/archives.data.ts]: Isn't this supposed to be consistent with the original article title?

**Review by Flouse — APPROVED** [review]

**Pull request merged**